README.md

```diff
@@ -93,7 +110,9 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
+```
+Referencing a function assigns its code and results to a variable or another function.
```

Comment:
Yes!! Calling a function is actually invoking that function to be executed immediately, while referencing is just referring to a particular function. I think you missed the second half of this question, which is asking to provide examples :) For your reference an example might be the following:
```
Example:
function sayHello(){
  console.log("Hello, world!");
}
// Calling a Function:
sayHello();
// Referencing a Function:
var button = document.querySelector("button");
button.addEventListener("click", sayHello);
```